static/chats.js

```diff
@@ -59,7 +59,7 @@ class Zinja {
   }
 
   formateRindu() {
-    this.vards = getCookie('name')
+    // this.vards = getCookie('name')
```

Comment:
Šo vajag?

Reply:
Man šķiet, ka šo var ņemt ārā.
Bet vajag faila augšā 3. rindā:
```js
var vards = getCookie('name') || "Viesis"
```

Reply:
Vēl smukāk būtu:
```js
let vards = getCookie('name') || 'Viesis'
```

Reply:
Aizkomentēto var ņemt ārā. Uztaisīju PR  un tad tikai pamanīju, ka neesmu izdzēsusi ārā. Gribēju, lai redzi, ko esmu uzskatījusi par lieku šajā koda fragmentā.

Reply:
Gribētu radināt, ka pull request ir galējais pieprasījums veikt izmaiņas, šajā gad. "master" zarā. Ja izstrāde ir procesā - var pielikt birku ("label") ar statusu, vai iekomentēt, ka vajag apskatīties.

Reply:
Izdzēšot rindu ārā es arī redzēšu, ka ir vēlme to dzēst, kas būtu pašsaprotamāks. Šajā gadījumā atliek vien minēt kāpēc kaut kas ir aizkomentēts.

Reply:
Jā, 3.rindā liekam let vards = getCookie('name') || 'Viesis'; bet es labāk gribētu let vards = 'Viesis' || getCookie('name')

Reply:
> Gribētu radināt, ka pull request ir galējais pieprasījums veikt izmaiņas, šajā gad. "master" zarā. Ja izstrāde ir procesā - var pielikt birku ("label") ar statusu, vai iekomentēt, ka vajag apskatīties.

Tieši tāpēc biju iedomājusies, ka vispirms apskatīsies manu jauno zaru, un tikai pēc tam veikšu PR. Turpmāk mācīšos likt birkas PR